doc/api/module.md

```diff
@@ -274,6 +274,7 @@ added:
 #### `new SourceMap(payload)`
```

Comment:
```suggestion
#### `new SourceMap(payload[, lineLengths])`
```

Reply:
> would it be more prudent to set the property with an option bag instead of a positional parameter?

No strong opinions here, but it's a pita to change this kind of thing later, so I'm feeling like it's probably a good idea, too.  Anyone else wanna weight on it?

Reply:
@bcoe would you mind taking a look at this PR? Thank you!